modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -627,7 +627,8 @@
 
 (defmethod unprepare/unprepare-value [:oracle ZonedDateTime]
   [_ t]
-  (format "timestamp '%s'" (t/format "yyyy-MM-dd HH:mm:ss.SSS VV" t)))
+  (format "timestamp '%s'" (-> (t/format "yyyy-MM-dd HH:mm:ss.SSS VV" t)
+                               (str/replace #" Z$" " UTC"))))
```

Comment:
`unprepare/unprepare-value [:oracle OffsetDateTime]` has the same problem.

Reply:
@metamben, I thought about that case while submitting your suggestion. My reasoning was as follows. Formatting `OffsetDateTime` for _some reason_ does not convert Z to 00:00. Hence the original workaround. But we expect other offsets to be as eg. `+13:00`, so it is fine to use just `#"Z$"` expression there.

`NZ` example:
```
(-> (t/format "yyyy-MM-dd HH:mm:ss.SSS ZZZZZ"
                (java.time.OffsetDateTime/now (java.time.ZoneId/of "NZ")))
      (str/replace #"Z$" "UTC"))
  ;; => "2024-01-17 07:57:25.306 +13:00"
```
I believe the current implementation of `unprepare/unprepare-value [:oracle OffsetDateTime]` is correct. Is there an error in my reasoning? Thanks.

Reply:
Your reasoning is correct, but I would still keep that symmetrical. It would still work, wouldn't be so expensive that it hurts, and we would get a clearer error on the off chance that another special case ending in `Z` gets introduced.

But it's not essential for sure, so I'm totally fine with your leaving it as is.

Reply:
I did as suggested. Methods' implementations are now symmetric.